src/cdk/app.ts

```diff
@@ -99,15 +94,15 @@ export class MiraApp {
   }
 
   static getBaseStackName (suffix?: string): string {
-    return getBaseStackName(suffix)
+    return MiraConfig.getBaseStackName(suffix)
```

Comment:
Maybe it worth to remove completely this method, and use directly the `MiraConfig.getBaseStackName`.
Or at least we can deprecate this one and remove it in the next version.